.github/workflows/release.yml

```diff
@@ -0,0 +1,63 @@
+name: Release
+
+on:
+  push:
+    branches: main
+  workflow_dispatch:
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+    env:
+      SHA: ${{ github.sha }}
+      GH_TOKEN: ${{ github.token }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - id: tag
+        name: Tag
+        shell: bash
```

Comment:
This is the default and can be removed. Oddly, it's required in `action.yml` composites, not normal workflow steps.